modules/olm-overriding-operator-pod-affinity.adoc

```diff
@@ -0,0 +1,307 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-scheduler-node-affinity.adoc
+// * nodes/scheduling/nodes-scheduler-pod-affinity.adoc
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+
+ifeval::["{context}" == "nodes-scheduler-pod-affinity"]
+:pod:
+endif::[]
+ifeval::["{context}" == "nodes-scheduler-node-affinity"]
+:node:
+endif::[]
+ifeval::["{context}" == "olm-adding-operators-to-a-cluster"]
+:olm:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="olm-overriding-operator-pod-affinity_{context}"]
+
+ifdef::olm[]
+= Controlling where an Operator is installed
+endif::olm[]
+
+ifdef::pod[]
+= Using pod affinity and anti-affinity to control where an Operator is installed
+endif::pod[]
+
+ifdef::node[]
+= Using node affinity to control where an Operator is installed
+endif::node[]
+
+By default, when you install an Operator, {product-title} installs the Operator pod to one of your worker nodes randomly. However, there might be situations where you want that pod scheduled on a specific node or set of nodes.
+
+The following examples describe situations where you might want to schedule an Operator pod to a specific node or set of nodes:
+
+* If an Operator requires a particular platform, such as `amd64` or `arm64`
+* If an Operator requires a particular operating system, such as Linux or Windows
+* If you want Operators that work together scheduled on the same host or on hosts located on the same rack
+* If you want Operators dispersed throughout the infrastructure to avoid downtime due to network or hardware issues
+
+ifdef::olm[]
+You can control where an Operator pod is installed by adding node affinity, pod affinity, or pod anti-affinity constraints to the Operator's `Subscription` object. Node affinity is a set of rules used by the scheduler to determine where a pod can be placed. Pod affinity enables you to ensure that related pods are scheduled to the same node. Pod anti-affinity allows you to prevent a pod from being scheduled on a node. 
+endif::olm[]
+
+ifdef::pod[]
+You can control where an Operator pod is installed by adding a pod affinity or anti-affinity to the Operator's `Subscription` object. 
+endif::pod[]
+
+ifdef::node[]
+You can control where an Operator pod is installed by adding a node affinity constraints to the Operator's `Subscription` object.
+endif::node[]
+
+ifdef::olm[]
+The following examples show how to use node affinity or pod anti-affinity to install an instance of the Custom Metrics Autoscaler Operator to a specific node in the cluster:
+endif::olm[]
+ifdef::node[]
+The following examples show how to use node affinity to install an instance of the Custom Metrics Autoscaler Operator to a specific node in the cluster:
+endif::node[]
+
+ifndef::pod[]
+.Node affinity example that places the Operator pod on a specific node
+[source,terminal]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  creationTimestamp: '2022-12-21T14:54:47Z'
```

Comment:
No need those metadata for the new subscription. The necessary args for the subscription, please help update all subscription samples, thanks!
```yaml
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: openshift-custom-metrics-autoscaler-operator
  namespace: openshift-keda
spec:
  name: my-package
  source: my-operators
  sourceNamespace: operator-registries
  config:
    affinity:
      nodeAffinity: <1>
        requiredDuringSchedulingIgnoredDuringExecution:
          nodeSelectorTerms:
          - matchExpressions:
            - key: kubernetes.io/hostname
              operator: In
              values:
              - ip-10-0-163-94.us-west-2.compute.internal
             ...
```